config/7.9.1/common.yaml

```diff
@@ -40,9 +40,6 @@ console:
               - name: "[[.ApplicationName]]-[[.Console.Name]]"
                 image: "[[.Console.ImageURL]]"
                 imagePullPolicy: Always
-                resources:
-                  limits:
-                    memory: 2Gi
```

Comment:
is console the only component this need to be removed from? What about kieservers, smart router, & pim?

Reply:
it should be remove on all places, except for the trial environment.

Reply:
Removed